src/cursor/cursor.ts

```diff
@@ -1068,6 +1061,11 @@ class Cursor extends CoreCursor {
   getLogger(): any {
     return this.logger;
   }
+
+  /** Clone the cursor */
+  clone(): Cursor {
+    return new Cursor(this.topology, this.operation, this.options);
```

Comment:
I see CommandCursor, AggregationCursor, and ChangeStreamCursor don't have clone overrides, I think this should be only implemented in CoreCursor, and take the form:
```ts
    clone() {
        return new this.constructor(/* ... */);
    }
```
I'm not sure what the return type would look like though

Reply:
type is `clone(): this`, but there's no way to dynamically call `this.constructor` without casting to `any` or other similar tricks. I modified it to do just that for now, we can revisit the decision when we turn linting on.